website/content/docs/developers/release-process-maven.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Publish Release to Maven Central
+---
+
+# Account Setup
+Following http://central.sonatype.org/ 
+
+1. Request access to publish to com.twitter by creating account and requesting access in https://issues.sonatype.org
+2. Twitter contact needed to confirm access (see example https://issues.sonatype.org/browse/OSSRH-22297)
+3. Install local GPG/PGP to sign files by following http://central.sonatype.org/pages/requirements.html#sign-files-with-gpgpgp 
+4. Publish public key (example http://central.sonatype.org/pages/working-with-pgp-signatures.html#distributing-your-public-key)
+5. Confirm public key is published (example http://pgp.mit.edu/ note leading ```0x``` required for hex keys)
+
+# Release Steps
+Follow [Manual Staging Bundle Creation and Deployment](http://central.sonatype.org/pages/manual-staging-bundle-creation-and-deployment.html) by signing files manually, steps summarized below.
+
+Two artifacts are required for each version: ```heron-api``` and ```heron-storm```
+Example POM files required located in ```heron/release/maven-central/```
+
+### Step 1 - Download and install ```heron-api-install-{VERSION}-{DIST}.sh``` from https://github.com/twitter/heron/releases
```

Comment:
The steps should instead describe how to create the maven artifacts using bazel, as opposed to downloading the artifacts from elsewhere.